src/preferences.py

```diff
@@ -7,6 +7,13 @@
 from graphs import file_io, graphs, misc, plot_styles, utilities
 
 
+MIGRATION_KEYS = [
+    # old -> new
+    ("handle_duplicates", "other_handle_duplicates"),
+    ("hide_unselected", "other_hide_unselected"),
+]
```

Comment:
This can be useful in the longer term. The current code handles outdated keys relatively well by simply using the template value, but this does keep user settings set at least as long as the migration key is defined.